tensorboard/webapp/runs/store/runs_selectors_test.ts

```diff
@@ -449,4 +449,41 @@ describe('runs_selectors', () => {
       });
     });
   });
+
+  describe('#getColorGroupRegexString', () => {
+    beforeEach(() => {
+      // Clear the memoization.
+      selectors.getColorGroupRegexString.release();
+    });
+
+    it('returns regex string when it is group by regex', () => {
+      const state = buildStateFromRunsState(
+        buildRunsState({
+          colorGroupRegexString: 'foo(\\d+)',
+          initialGroupBy: {key: GroupByKey.RUN},
+          userSetGroupByKey: GroupByKey.REGEX,
```

Comment:
If these are irrelevant to the spec, omit it. It makes the test harder to maintain when specs have too many unnecessary setup code. Here and below.